src/modes/hints/client/activate.js

```diff
@@ -85,6 +85,16 @@ const activators = {
   focusLink: (event, target) => {
     target.focus()
     return 'Reset'
+  },
+  yankLink: (event, target) => {
+    const copyText = copyEvent => {
```

Comment:
Any reason you're implementing a new copy function instead of using the one over in https://github.com/lusakasa/saka-key/blob/master/src/lib/dom.js#L143?